packages/airnode-validator/src/config/config.ts

```diff
@@ -153,23 +153,51 @@ export const chainOptionsSchema = z
   })
   .strict();
 
+// Authorizations
```

Comment:
I am not sure what this comment want to say.

Reply:
This comment (and the below similar ones) were meant as code dividers but I realize they're confusing so I'll remove.